### DISCUSSION TITLE
Apkmirror disabled by default?

### DISCUSSION THREAD

**Discussion by archon810** [rumboalla/apkupdater#531] [General]
Here https://github.com/rumboalla/apkupdater/commit/72ef10744a21a3d601ac50661edd7b3af4d75bf6

@rumboalla Can you please explain why? Is there an issue?

**Comment by rumboalla** [comment]
> Here [72ef107](https://github.com/rumboalla/apkupdater/commit/72ef10744a21a3d601ac50661edd7b3af4d75bf6)
> 
> @rumboalla Can you please explain why? Is there an issue?

As it is, the ApkMirror source provides a poor experience. The lack of direct downloads, the walls of ads and forcing all the other users of the app to have 1h late notifications...This is why I've made it opt-in. Anyone who wants to use the source is one click away from it.

I do understand the servers are not free, and you have to make a living. If you have any idea on how to make this better, I'm open to suggestions.
